package.json

```diff
@@ -34,14 +34,6 @@
     "release": "release-it --verbose",
     "release:alpha": "release-it --verbose --preRelease=alpha"
   },
-  "peerDependencies": {
-    "@chakra-ui/react": "^1.8.7 || 2.x",
-    "@emotion/react": "^11.8.2",
-    "@emotion/styled": "^11.8.1",
-    "react": "^16.8.0 || 17.x || 18.x",
-    "react-dom": "^16.8.0 || 17.x || 18.x",
-    "swr": "^2.0.3"
```

Comment:
I think these should stay as peer dependencies rather than production dependencies because it allows the dependency to be satisfied by an installing repo having a similar version

Reply:
I do think we can keep these, but they should be updated to require react 18 and chakra v2. Using chakra v2 in the consuming app while the web reader was on chakra v1 was causing some very strange styling issues.

Reply:
Oh yes true. The updates were good, just should stay peer instead of production dependencies